services/dqa-secondary-reports/main.py

```diff
@@ -305,6 +343,14 @@ def index():
                              dqa_resource.serve_table_totals(version, table),
                              methods=['GET'])
 
+            for site in dqa_resource.site_totals[version]:
+                app.add_url_rule(('/pedsnet/' + version + '/table-totals/' +
```

Comment:
It would be more clear at a glance what the path and route name are if they are defined as format-able strings above the loop:

```python
urlf = '/pedsnet/{version}/table-totals/{table}/{site}/'
namef = '{version}_table-totals_{table}_{site}'

for site in dqa_resource.site_totals[version]:
    app.add_url_url(urlf.format(version=version, table=table, site=site),
                    namef.format(version=version, table=table, site=site),
                    ...)
```